sharing/mainfunction.py

```diff
@@ -0,0 +1,994 @@
+   'reportDate': Timestamp('2021-01-30 00:00:00'),
+   'reporterID': 'Reporter R3',
+   'sampleID': 'Sample S102',
+   'type': 'nPMMoV',
+   'typeOther': nan,
+   'uWwMeasureID': 'Measure WW102',
+   'unit': 'gcMl',
+   'unitOther': 'gcmnPMMoV',
+   'value': 72000},
+  {'analysisDate': Timestamp('2021-01-26 00:00:00'),
+   'assayID': 'Assay Y103',
+   'fractionAnalyzed': 'liquid',
+   'index': 2,
+   'instrumentID': 'Instrument IN203',
+   'labID': 'Lab L103',
+   'notes': 'nPMMoV measure linked to CovidN2 by index',
+   'reportDate': Timestamp('2021-01-26 00:00:00'),
+   'reporterID': 'Reporter R4',
+   'sampleID': 'Sample S103',
+   'type': 'nPMMoV',
+   'typeOther': nan,
+   'uWwMeasureID': 'Measure WW103',
+   'unit': 'gcMl',
+   'unitOther': 'gcmnPMMoV',
+   'value': 92000},
+  {'analysisDate': Timestamp('2021-01-28 00:00:00'),
+   'assayID': 'Assay Y104',
+   'fractionAnalyzed': 'solid',
+   'index': 2,
+   'instrumentID': 'Instrument IN204',
+   'labID': 'Lab L104',
+   'notes': 'nPMMoV measure linked to CovidN2 by index',
+   'reportDate': Timestamp('2021-01-26 00:00:00'),
+   'reporterID': 'Reporter R5',
+   'sampleID': 'Sample S104',
+   'type': 'nPMMoV',
+   'typeOther': nan,
+   'uWwMeasureID': 'Measure WW104',
+   'unit': 'gcMl',
+   'unitOther': 'gcmnPMMoV',
+   'value': 102000},
+  {'analysisDate': Timestamp('2021-02-26 00:00:00'),
+   'assayID': 'Assay Y105',
+   'fractionAnalyzed': 'solid',
+   'index': 6,
+   'instrumentID': 'Instrument IN205',
+   'labID': 'Lab L105',
+   'notes': 'nPMMoV measure linked to CovidN2 by index6',
+   'reportDate': Timestamp('2021-01-31 00:00:00'),
+   'reporterID': 'Reporter R6',
+   'sampleID': 'Sample S105',
+   'type': 'nPMMoV',
+   'typeOther': nan,
+   'uWwMeasureID': 'Measure WW105',
+   'unit': 'gcMl',
+   'unitOther': 'gcmnPMMoV',
+   'value': 145000},
+  {'analysisDate': Timestamp('2021-02-25 00:00:00'),
+   'assayID': 'Assay Y106',
+   'fractionAnalyzed': 'solid',
+   'index': 2,
+   'instrumentID': 'Instrument IN206',
+   'labID': 'Lab L106',
+   'notes': 'nPMMoV measure linked to CovidN2 by index',
+   'reportDate': Timestamp('2021-01-25 00:00:00'),
+   'reporterID': 'Reporter R7',
+   'sampleID': 'Sample S106',
+   'type': 'nPMMoV1',
+   'typeOther': nan,
+   'uWwMeasureID': 'Measure WW106',
+   'unit': 'gcM',
+   'unitOther': 'gcmnPMMoV1',
+   'value': 89000},
+  {'analysisDate': Timestamp('2021-02-08 00:00:00'),
+   'assayID': 'Assay Y107',
+   'fractionAnalyzed': 'solid',
+   'index': 6,
+   'instrumentID': 'Instrument IN207',
+   'labID': 'Lab L107',
+   'notes': 'nPMMoV measure linked to CovidN2 by index',
+   'reportDate': Timestamp('2021-01-26 00:00:00'),
+   'reporterID': 'Reporter R8',
+   'sampleID': 'Sample S107',
+   'type': 'nPMMoV',
+   'typeOther': nan,
+   'uWwMeasureID': 'Measure WW107',
+   'unit': 'gcMl',
+   'unitOther': 'gcmnPMMoV',
+   'value': 98000}]}
+
+organization = "PHAC"
+
+def create_dataset(rule,data=dataset, org= organization):
```

Comment:
Can we rename this to `create_shared_dataset`?

The parameter names should follow whats defined README file in the `sharing_functions` branch.

Why are the parameters being defaulted to the variables you defined above?